DPG/Tasks/AOTTrack/qaEfficiency.cxx

```diff
@@ -591,9 +590,12 @@ struct QaEfficiency {
 
   void initMC(const AxisSpec& axisSel)
   {
-    if (!doprocessMC) {
+    if (!doprocessMC && !doprocessMCWithoutCollisions) {
```

Comment:
I'd add also here a `LOG(fatal)`, so that one can understand immediately that he/she needs to switch on one of the `process` functions